packages/experimental/NativeDatePicker/ios/DatePickerDelegate.swift

```diff
@@ -0,0 +1,22 @@
+import FluentUI
+
+class DatePickerDelegate: DateTimePickerDelegate {
+    weak var manager: DatePickerManager?
+    let picker: DateTimePicker
+    let didPickBlock: RCTResponseSenderBlock
+
+    public init(manager: DatePickerManager, picker: DateTimePicker, didPickBlock: @escaping RCTResponseSenderBlock) {
+        self.manager = manager
+        self.picker = picker
+        self.didPickBlock = didPickBlock
+    }
+
+    public func dateTimePicker(_ dateTimePicker: DateTimePicker, didPickStartDate startDate: Date, endDate: Date) {
+        let formatter = ISO8601DateFormatter()
+        formatter.formatOptions.insert(.withFractionalSeconds)
```

Comment:
why do we need this?

Reply:
Why do we need the function, or why do we need the ISO8601DateFormatter?  The former is the callback from the FluentUI-Apple class, which is how we communicate the picked values.  The latter is to convert a date into a string, since we can't pass a Date object directly back to RN.

Reply:
If the question is why we need `.withFractionalSeconds`, it's because JS doesn't understand how to parse an ISO 8601 date without that part.  I did write a helper function to manually parse dates, but with this the built-in Date constructor can also do it (though I read it was ['strongly discouraged'](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date) to use that)

Reply:
Yeah sorry I meant the fractional seconds bit. 
The fact that you are discouraged from using the constructor is... unfortunate. I'm not a fan of doing manual parsing of dates, but it seems we might have to if we don't want to add a new dependency / use the base one? Wow, that's annoying

Reply:
I figure if somebody consuming this module wants to just use the constructor, I am ensuring it probably works fine.  But given the warning I wanted to have an example of doing it manually too.

Reply:
Ah, this is example code. That is a good distinction to make, thank you!